tutorials/plantUml.md

```diff
@@ -278,3 +278,51 @@ package "Rule Of Thumb";{
 
 Explicitly define all symbols to avoid any potential layout mishaps.
 </box>
+
+### Using reference frames
+
+<table>
+<colgroup>
+<col style="width: 70%" />
+<col style="width: 30%" />
+</colgroup>
+
+<thead>
+<tr class="header">
+  <th>Source</th>
+  <th>Result</th>
+</tr>
+</thead>
+
+<tbody>
+
+<tr class="odd">
+<td>
+<pre>
+ref over MSLogic, TextUi : get minefield appearance
+</pre>
+</td>
+<td>
+<img src="images/plantuml/ParentReferenceFrameDiagram.png">
+</td>
+</tr>
+
+<tr class="even">
+<td>
+<pre>
```

Comment:
Agree with @NeoHW, a suggestion would be to label the respective ref/sd frames to indicate clearly which is which for more structure. Just a minor comment.